docs/tutorials/realistic_sims/noise.md

```diff
@@ -69,16 +98,45 @@ print(f"noiseless = {noiseless_exp}") # markdown-exec: hide
 print(f"noisy = {noisy_exp}") # markdown-exec: hide
 ```
 
+## Analog noisy simulation
+
+At the moment, analog noisy simulations are only compatable with the Pulser backend.
+```python exec="on" source="material-block" session="noise" result="json"
+from qadence import DiffMode, NoiseHandler, QuantumModel
+from qadence.blocks import chain, kron
+from qadence.circuit import QuantumCircuit
+from qadence.operations import AnalogRX, AnalogRZ, Z
+from qadence.types import PI, BackendName, NoiseProtocol
+
+
+analog_block = chain(AnalogRX(PI / 2.0), AnalogRZ(PI))
+observable = Z(0) + Z(1)
+circuit = QuantumCircuit(2, analog_block)
+
+options = {"noise_probs": 0.1}
+noise = NoiseHandler(protocol=NoiseProtocol.ANALOG.DEPOLARIZING, options=options)
+model_noisy = QuantumModel(
+    circuit=circuit,
+    observable=observable,
+    backend=BackendName.PULSER,
+    diff_mode=DiffMode.GPSR,
+    noise=noise,
+)
+noisy_expectation = model_noisy.expectation()
+print(f"noisy = {noisy_expectation}") # markdown-exec: hide
+```
+
+
 ## Digital noisy simulation
 
-When dealing with programs involving only digital operations, several options are made available from [PyQTorch](https://pasqal-io.github.io/pyqtorch/latest/noise/) via the `NoiseType`. One can define noisy digital operations with `DigitalNoise`as follows:
+When dealing with programs involving only digital operations, several options are made available from [PyQTorch](https://pasqal-io.github.io/pyqtorch/latest/noise/) via the `NoiseProtocol.DIGITAL`. One can define noisy digital operations as follows:
 
 ```python exec="on" source="material-block" session="noise" result="json"
-from qadence import NoiseType, DigitalNoise, RX, run
+from qadence import NoiseProtocol, RX, run
 import torch
 
-noise = DigitalNoise(NoiseType.BITFLIP, error_probability = 0.2)
-noise = DigitalNoise.bitflip(error_probability = 0.2) # equivalent
+noise = NoiseHandler(NoiseProtocol.DIGITAL.BITFLIP, {"error_probability": 0.2})
+noise = NoiseHandler.bitflip({"error_probability": 0.2}) # equivalent
```

Comment:
Would it make sense to build on this to chain noise models ? Smtg like:

```python
noise = NoiseHandler.bitflip(...).phaseflip(...).<other-noise-model>(...)
```

That keep adding in the list.

Reply:
Originally I intented for these methods to instantiate easily a handler with one type of noise. Maybe I could add a `append` method to add to the list?

Reply:
Added the `append`